apps/vh/base/nonprod-role.yaml

```diff
@@ -12,6 +12,9 @@ rules:
 - apiGroups: ['helm.fluxcd.io']
   resources: ['helmreleases']
   verbs: ['delete']
+- apiGroups: ['v2beta1.helm.fluxcd.io']
```

Comment:
this definitely right and not?
```suggestion
- apiGroups: ['helm.fluxcd.io']
```

Reply:
I was looking at the output of `k get apiservices.apiregistration.k8s.io -A`. There's v1.helm.fluxcd.io and v2beta1.helm.fluxcd.io. Will helm.fluxcd.io do it for both?

Reply:
I think so

Reply:
I've reverted it there